cpp/src/Ice/SSL/OpenSSLTransceiverI.cpp

```diff
@@ -36,10 +36,13 @@ extern "C"
 
 namespace
 {
-    std::function<bool(bool, X509_STORE_CTX* ctx, const Ice::SSL::ConnectionInfoPtr&)>
-    createDefaultVerificationCallback()
+    bool defaultVerificationCallback(bool ok, X509_STORE_CTX*, const Ice::SSL::ConnectionInfoPtr&) { return ok; }
+
+    SSL_CTX* defaultSSLContextSelectionCallback(const string&)
```

Comment:
We didn't have a default context. This new callback is used when the user doesn't provide one and ensures the default context would use the default platform CAs.